src/app/pages/offline-map/offline-map.page.html

```diff
@@ -7,10 +7,32 @@
   </ion-toolbar>
 </ion-header>
 <ion-content>
-  <ion-button expand="full">
-    <input type="file" (change)="loadFile($event.target.files)" id="file-input"
-      accept="application/zip">
-  </ion-button>
+  <div class="desc">
+    <div>
+      <ion-text color="primary">
+        <h1>Nedlastning av kartpakker</h1>
+        <p>Nedlastning av kartpakker foregår i to trinn:</p>
+        <ol>
+          <li>Gå inn på nedlastningssiden og last ned kartpakke til telefonen.</li>
+          <li>Trykk på "Velg fil"-knappen nedenfor og velg nedlastet zip-fil for å installere kartpakken.</li>
+        </ol>
+      </ion-text>
+    </div>
```

Comment:
Supert med så fin beskrivelse!
Burde vi skrevet at offline-kart kun funker i app?
Jeg synes det er fint at menyvalget er tilgjengelig på web også, så kan vi bruke det som "reklame" for appen.

Reply:
Det kan vi legge til senere tenker jeg

Reply:
supert!